### PR TITLE
Simple misspelling typos...

### PR DESCRIPTION
**Pull request by stratosgear**
Simple misspelling typos...

### CODE REVIEW

File: doc/DOCUMENTATION.org [hunk 1]
```diff
@@ -2548,7 +2548,7 @@ Text related commands (start with ~x~):
 | ~SPC x a ;~ | align region at ;                                             |
 | ~SPC x a =~ | align region at =                                             |
 | ~SPC x a a~ | align region (or guessed section) using default rules         |
-| ~SPC x a c~ | align current intendation region using default rules          |
+| ~SPC x a c~ | align current intentation region using default rules          |
```
Comment (d12frosted) [inline doc/DOCUMENTATION.org]: shouldn't it be 'indentation'? :D

**Comment by d12frosted** [comment]
Please, squash your commits 😸

**Comment by stratosgear** [comment]
Squashed!

**Comment by d12frosted** [comment]
Thank you :+1: Cherry-picked into `develop`, you can safely delete your branch.

**Pull request merged**